src/openms/include/OpenMS/ANALYSIS/ID/PeptideIndexing.h

```diff
@@ -376,8 +383,17 @@ namespace OpenMS
         OPENMS_LOG_INFO << "Building trie ...";
         StopWatch s;
         s.start();
-        AhoCorasickAmbiguous::FuzzyACPattern pattern;
-        AhoCorasickAmbiguous::initPattern(pep_DB, aaa_max_, mm_max_, pattern);
+        auto s1 = std::chrono::high_resolution_clock::now();
+
+
+        //AhoCorasickAmbiguous::FuzzyACPattern pattern;
+        //AhoCorasickAmbiguous::initPattern(pep_DB, aaa_max_, mm_max_, pattern);
+        Size pos = 0;
+        Size idx = 0;
+        AhoCorasickDA fuzzyAC(pep_DB);
+        auto s2 = std::chrono::high_resolution_clock::now();
+        std::chrono::duration<double,std::milli> elapsed = s2 - s1;
+        std::cout << "Construction time " << elapsed.count() << " ms"<< std::endl;
```

Comment:
why the extra chrono?
StopWatch is a high-resolution clock
(just don't convert to `int()` as done below to retain its accuracy)